compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -416,23 +422,33 @@ static LogicalResult setSandboxRootConfig(func::FuncOp entryPointFn,
   int64_t nLoops = linalgOp.getNumLoops();
   if (nLoops >= 3) {
     parallelTileSizes.append(nLoops - 3, 1);
-    parallelTileSizes.push_back(getMaxTileSize(
-        0, flowTileSizes[nLoops - 3], target2ndTileSizes[0], vectorSize));
+    parallelTileSizes.push_back(
+        disableMatmulPadPipeline
+            ? getMaxTileSize(-0, flowTileSizes[nLoops - 3],
```

Comment:
`-0`? :)

Reply:
haha, good catch. I copied it from `git diff` and did not notice that `-` was not deleted.